### PR TITLE
Fix analyzer warning.

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

**Comment by pylaligand** [comment]
Not sure how the presubmit build was happy with it in the first place...

**Comment by pylaligand** [comment]
Ah, that's because it didn't. The green check I saw was the CLA...

File: packages/flutter_tools/bin/fuchsia_builder.dart [hunk 1]
```diff
@@ -28,7 +28,7 @@ const List<String> _kRequiredOptions = const <String>[
   _kOptionWorking,
 ];
 
-void main(List<String> args) async {
+void main(List<String> args) {
```
Comment (johnmccutchan) [inline packages/flutter_tools/bin/fuchsia_builder.dart]: Future<Null> main(List<String> args) async {
...}

Comment (pylaligand) [inline packages/flutter_tools/bin/fuchsia_builder.dart]: Does main need to be async?

**Comment by pylaligand** [comment]
Abandoning in favor of flutter/flutter#7073.

**Pull request closed without merging**